src-built-in/components/advancedAppCatalog/src/stores/storeActions.js

```diff
@@ -235,8 +235,28 @@ async function addApp(id, cb = Function.prototype) {
 		delete appConfig.window.url;
 	}
 
-	if (typeof app.manifest !== "object") {
-		appConfig.manifest = { ...appConfig };
+	let manifest;
+	if (app.manifestType === "Finsemble") {
+		try {
+			//Attempt to parse a string as JSON
```

Comment:
Nit: Unneeded comment

Reply:
I'm going to leave these comments, since it is not obvious why I would be doing these two things on a string property. If anything, they should be fleshed out so the next person to come along and look at this code doesn't need to have a comprehensive understanding of the FDC3 spec